x/observer/migrations/v11/migrate.go

```diff
@@ -0,0 +1,62 @@
+package v11
+
+import (
+	"slices"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/zeta-chain/node/pkg/chains"
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+type observerKeeper interface {
+	GetChainParamsList(ctx sdk.Context) (val types.ChainParamsList, found bool)
+	SetChainParamsList(ctx sdk.Context, chainParams types.ChainParamsList)
+}
+
+var disableBlockScanChainIDs = []int64{
+	// localnet
+	chains.GoerliLocalnet.ChainId,
+
+	// testnets
+	chains.AvalancheTestnet.ChainId,
+	chains.ArbitrumSepolia.ChainId,
+	chains.BscTestnet.ChainId,
+	chains.BaseSepolia.ChainId,
+	chains.Amoy.ChainId,
+
+	// mainnets
+	// the other mainnets will be deprecated later
```

Comment:
Does this need to be tracked via an issue ?

Reply:
Issue linked